fenix/app/src/main/res/layout/search_selector.xml

```diff
@@ -13,7 +13,7 @@
         android:layout_width="40dp"
         android:layout_height="28dp"
         android:layout_marginHorizontal="8dp"
-        app:cardBackgroundColor="@color/photonWhite"
+        app:cardBackgroundColor="@color/fx_mobile_layer_color_2"
```

Comment:
Could you double check if this behaves correctly in private theme? `fx_mobile_layer_color_2` isn't defined in private theme and we would use `?attr/layer2` to account for that. I